lib/matplotlib/projections/polar.py

```diff
@@ -1161,34 +1155,78 @@ def set_theta_direction(self, direction):
     def get_theta_direction(self):
         """
         Get the direction in which theta increases.
-
         -1:
            Theta increases in the clockwise direction
-
         1:
            Theta increases in the counterclockwise direction
         """
         return self._direction.get_matrix()[0, 0]
 
     def set_rmax(self, rmax):
+        """Updates the maximum radius
```

Comment:
"Update the maximum radius."

[PEP-257](https://www.python.org/dev/peps/pep-0257/):

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".